src/components/content-tab-overview.tsx

```diff
@@ -152,7 +152,7 @@ function ShortcutsSection( { selectedSite }: Pick< ContentTabOverviewProps, 'sel
 	if ( installedApps.vscode ) {
 		// Use VS Code as a default even if none of the editors are installed
 		buttonsArray.push( {
-			label: 'VS Code',
+			label: __( 'VS Code' ),
```

Comment:
As outlined in https://github.com/Automattic/studio/pull/48#issuecomment-2081362956, we explicitly changed this string and `PhpStorm` to be not translatable. Based on their webpages, seems the IDE's names are not translated.

Reply:
Should we also add a note for translators for `VS Code` and `PhpStorm` phrases?

Reply:
Ok, I see. In that case, I wonder if we could add a comment for translators to omit the IDE's name. Similar to what we added [here](https://github.com/Automattic/studio/blob/4f373577e40cae8af9fa20275d77e3f8ac3ea3f3/src/components/content-tab-overview.tsx#L164). WDYT?

Reply:
Makes sense, updated by 400e186.